Makefile.standard_app

```diff
@@ -105,10 +111,8 @@ ifneq ($(DISABLE_STANDARD_WEBUSB), 1)
     DEFINES += HAVE_WEBUSB WEBUSB_URL_SIZE_B=$(WEBUSB_URL_SIZE_B) WEBUSB_URL=$(WEBUSB_URL)
 endif
 
-ifneq ($(DISABLE_STANDARD_BAGL_UX_FLOW), 1)
-ifneq ($(TARGET_NAME), TARGET_STAX)
-    DEFINES += HAVE_UX_FLOW
```

Comment:
I' think this one is useful for some apps on Nanos (but should not be put on all apps).